index.Rmd

```diff
@@ -29,9 +29,9 @@ kable(head(walkroutes_ta), align = "c") %>%
   kable_styling(bootstrap_options = "bordered", full_width = F)
 ```
 
-According to the policy of the 15-Minute Community-Life Circle, a 15-min walk to parks is important for residents in China, which corresponds to about a 1000-m distance. I called the travel navigation function of online map to scrape recommended walking routes from residential buildings to the entrances of Xuanwu Lake Park, with a 15-min duration as the threshold. This data also includes the information of route distance, euclidean distance, the number of turns, the number of crossings,      starting, and ending points.
```

Comment:
Would again be great to have in text citations here :) Which online map did you use? Google maps? How did you decide on a 15 minute threshold? After the word "crossings", I think there are too many spaces.